tests/testJacobian.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008-2013 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+import numpy
+import unittest
+
+import lsst.daf.base as dafBase
+import lsst.afw.image as afwImage
+import lsst.utils.tests
+import lsst.meas.base.tests
+from lsst.meas.base.tests import (AlgorithmTestCase, SingleFramePluginTransformSetupHelper)
+
+class JacobianTestCase(AlgorithmTestCase):
+    def setUp(self):
+        self.center = lsst.afw.geom.Point2D(50.1, 49.8)
+        self.bbox = lsst.afw.geom.Box2I(lsst.afw.geom.Point2I(-20, -30),
+                                        lsst.afw.geom.Extent2I(140,160))
+        self.dataset = lsst.meas.base.tests.TestDataset(self.bbox)
+        self.dataset.addSource(100000.0, self.center)
+
+        md = dafBase.PropertyList()
+
+        for k, v in (
```

Comment:
For the purposes of the test, you could omit some of these keys -- you can drop `EQUINOX`, `RADESYS`, or `CDi_j` (or `CDELTn`) without changing the logic. Less code → fewer mistakes.